server/appServices.js

```diff
@@ -191,17 +193,20 @@ async function watchIosVariants(namespace, kubeclient) {
  * @param {String} namespace - The namespace to watch
  * @param {*} kubeclient - kubernetes client
  */
-async function watchMobileSecurityConfigMaps(namespace, kubeclient) {
-  const mssConfigMapStream = await kubeclient.api.v1.watch.namespace(namespace).configmaps.getObjectStream();
-  mssConfigMapStream.on('data', event => {
+async function watchMobileSecurityApp(namespace, kubeclient) {
```

Comment:
```suggestion
async function watchMobileSecurityApps(namespace, kubeclient)
```

You are watching **all** apps, so the function should indicate so.